### DISCUSSION TITLE
Yi Dome Guard IFUSY36S4 R30GB

### DISCUSSION THREAD

**Discussion by DoSpamu** [roleoroleo/yi-hack-Allwinner-v2#166] [Q&A]
Hello, it's any working version for Dome Guard IFUSY36S4G3LOC200907 
R30GB US V1.8 
FW 9.0.05.33

**Comment by roleoroleo** [comment]
Yes, it's released but in beta version.
See the readme.

**Reply by DoSpamu** [reply]
I try install r30gb but nothing happens. 
I format SDcard from yi home app, delete all files. 
Copy .tgz - nothing
Copy .tar - nothing
Copy unpacked .tar - nothing.

**Reply by DoSpamu** [reply]
OK, nevermind. I copied all unpacked files from .tar (factory, newbackup, yi-hack, lower_half_init) to formatted SD card (but formated not in Windows, not by Yi Home App, but formated on Android phone, i haved the same problem with YI Action Cam when not recordfiles and can't update her)

**Reply by roleoroleo** [reply]
Please explain me better.

**Reply by AndWalu** [reply]
> Yes, it's released but in beta version.
> See the readme.

I bought YI Dome Guard Camera, SN: IFUSY36OO38NO...
  https://www.aliexpress.com/item/4000674706324.html
Serial console is connected.
If you want, I can help with testing. 
r30gb almost works.
- High resolution snapshot causes restart.
- There is no IP in WWW interface if wired connection is used.
- Events playing doesn't work in WWW interface.

**Reply by WalkingDot** [reply]
>     * High resolution snapshot causes restart.

It could be out of memory. Have you enabled swap? 

>     * Events playing doesn't work in WWW interface.

Could be a webbrowser problem.

My r30gb doesn't have these problems

**Reply by AndWalu** [reply]
Swap is an evil. ;)
Events player is now working, probably it was a problem with wired IP.
